lib/balena-lint.ts

```diff
@@ -198,7 +198,7 @@ const lintTsFiles = async function(
 	return errorReport.errorCount === 0 ? 0 : 1;
 };
 
-const lintMochaTestFiles = async function(files: string[]): Promise<number> {
+const lintMochaTestFiles = async function (files: string[]): Promise<number> {
```

Comment:
I'm only sad about this added space, since imo it makes generic function declarations look weird:
```ts
function <T>(value: T) {};
```
Too bad (again imo) that they don't offer a setting for this.
PS: But no one should care since we have autoformatting :)